index.html

```diff
@@ -827,12 +827,12 @@
 
 <p>This attribute is optional. User agents that don't have this attribute
 available must set it as undefined. When this attribute is available, if the
-<a href="https://www.w3.org/TR/html5/infrastructure.html#url-scheme">scheme</a> of the
+<a href="https://url.spec.whatwg.org/#concept-url-scheme">scheme</a> of the
 current page is <a href="http://tools.ietf.org/html/rfc2817">HTTPS</a>, this
 attribute must return the time immediately before the user agent starts the
 handshake process to secure the current connection. If this attribute is
 available but HTTPS is not used, this attribute must return zero.
-</p>
+</p>[[URL]]

```

Comment:
`[[URL]]` starts a new paragraph here. IMHO it should be inserted directly after "scheme", since it's more consistent with other references (such as RFC2616, RFC7230, and RFC7234) in this spec.